src/WebJobs.Script.WebHost/Scale/TableEntityConverter.cs

```diff
@@ -3,27 +3,27 @@
 
 using System;
 using System.Collections.Generic;
-using Microsoft.Azure.Cosmos.Table;
+using Azure;
+using Azure.Data.Tables;
 using Newtonsoft.Json.Linq;
 
 namespace Microsoft.Azure.WebJobs.Script.WebHost.Scale
 {
     /// <summary>
-    /// Class providing methods to convert between DynamicTableEntity and custom pocos
+    /// Class providing methods to convert between DynamicTableEntity and custom pocos.
     /// </summary>
     internal static class TableEntityConverter
     {
-        public static DynamicTableEntity ToEntity(object o,
+        public static TableEntity ToEntity(object o,
```

Comment:
Just to confirm - the new SDK you're moving to doesn't have any built ins to handle POCO conversions? I added this converter class because there wasn't before, and we need a way to convert to/from POCOs that don't implement ITableEntity